src/app/(sidebar)/transaction/sign/components/Overview.tsx

```diff
@@ -627,6 +788,82 @@ export const Overview = () => {
                 errorMsg={exErrorMsg}
               />
             </Box>
+
+            <Box gap="md" addlClassName="PageBody__content">
+              <Label size="md" htmlFor="">
+                Add a signature
+              </Label>
+
+              <>
+                {sigInputs.map((_, idx) => (
+                  <Box gap="xs" key={`${idx}-tx-sig`}>
+                    <PubKeyPicker
+                      id={`${idx}-tx-sig-pubkey`}
+                      placeholder="Public key"
+                      label=""
+                      value={sigInputs[idx]?.publicKey}
+                      error={sigInputsError[idx]?.publicKey}
+                      onChange={(e) =>
+                        handleSignatureOnChange(
+                          e.target.value,
+                          "publicKey",
+                          idx,
+                        )
+                      }
+                    />
+
+                    <TextPicker
+                      id={`${idx}-tx-sig-b64sig`}
+                      placeholder="B64 encoded signature"
```

Comment:
What does b64 mean? I thought it might mean base64, but in the screenshot looks like it is hex.

If it's base64, I think we should use the full term:
```suggestion
                      placeholder="Base64 encoded signature"
```

If it's hex, I think we should use a descriptive term:
```suggestion
                      placeholder="Hex encoded 64-byte ed25519 signature"
```

Reply:
It is hex. Thanks, @leighmcculloch !